lettuce/io.py

```diff
@@ -31,21 +31,22 @@ def write_png(filename, array2d):
 
 
 #def write_vtk(filename, res, ux, uy, p, t):
-def write_vtk(filename, point_dict, id):
-
-    vtk.gridToVTK("/Users/mariobedrunka/Downloads/data/" + filename + "_" + str(id),
+def write_vtk(point_dict, id, filename_base="./data/", filename="output"):
```

Comment:
`filename_base` is OK, `filename` is not needed
You can just pass the whole thing including path as `filename_base` (`filename_base="./data/output").

Also, if you want to write this to a data directory by default, please make sure that the directory exists:
```
directory = os.path.dirname(filename_base)
if not os.path.isdir(directory):
    os.mkdir(directory)
```